docs/source/markdown/options/pull.image.md

```diff
@@ -4,9 +4,9 @@
 ####> are applicable to all of those.
 #### **--pull**=*policy*
 
-Pull image policy. The default is **always**.
+Pull image policy. The default is **missing**.
```

Comment:
That doesn't look right, so let's bikeshed a bit more.

"missing" means Podman will not pull the image when it's in the storage.  "always" means Podman will attempt to pull the image - ignoring the storage - but may skip layers if they're already present.

"always" is the current behavior.

Reply:
Note that the defaults may differ between the various commands.

Reply:
So `podman pull` defaults to "always".  `podman run` defaults to "missing" etc.

Reply:
So I think we need to remove mentioning the default in this shared file and mention it at the include sites of the specific commands.

Reply:
see the top line in this file, these docs here are only for podman build and farm build NOT run and create

Reply:
Thanks! LGTM